src/Workspaces/VisualBasicTest/Formatting/FormattingTests.vb

```diff
@@ -812,16 +812,16 @@ End Class</Code>
             Dim expected = <Code>Class C
     Sub Method(Optional ByVal i As Integer = 1)
         Dim aa = 1 + _
-                 2 + _
-                 3
+                     2 + _
+                         3
```

Comment:
this formatting doesn't seem desirable.

Reply:
It is not desirable and I would love to fix it but I don't see a rule that would format it correctly and doesn't undo all the newer formatting. I have experimented with the available rules and you either get this behavior or the "2" is left aligned sometimes and correctly aligned others depending on what the line looks like before the formatting.

Reply:
The code that does the indenting based on the rules needs to understand " _ ' Optional Comment' and I have not found it yet or I need to use a different rule.

Reply:
> I would love to fix it but I don't see a rule that would format it correctly and doesn't undo all the newer formatting

What do you mean by 'newer formatting'?

> The code that does the indenting based on the rules needs to understand " _ ' Optional Comment' and I have not found it yet or I need to use a different rule.

Can you write a set of tests that show the behavior you want roslyn to have, but doesn't have?  That way we can then have a code change that makes those tests pass, while not regressing these other scenarios?

Reply:
That is my goal, I restored all the original tests, some fail is expected other fail because my code incorrectly breaks them. Is there a way to expect the old test to fail and have it pass?